api/clients/v2/config.go

```diff
@@ -0,0 +1,33 @@
+package v2
+
+import (
+	"github.com/Layr-Labs/eigenda/api/clients/codecs"
+	"time"
+)
+
+// VerificationMode is an enum that represents the different ways that a blob may be encoded/decoded between
+// the client and the disperser.
+type VerificationMode uint
+
+const (
+	// TODO: write good docs here for IFFT and NoIFFT (I need to update my understanding to be able to write this)
```

Comment:
It looks the "PointVerificationMode" field in config already documents what these two mean.

Actually, I am not entirely sure if we want to call out "how" (i.e. IFFT and then FFT) at the interface level. What about BlobFormat or something (point v.s. coeffs) that describes the blobs, not how they are processed underneath? cc @samlaf

Reply:
> It looks the "PointVerificationMode" field in config already documents what these two mean.

There's a description there, but I'm personally a bit confused by what it says, so I want to clarify to be able to write a good description here.

> What about BlobFormat or something

Makes sense, I'll defer to you and Sam for the proper name here